scripts/functest.sh

```diff
@@ -40,6 +40,12 @@ for preference in $(${KUBECTL} get virtualmachineclusterpreferences --no-headers
         echo "functest failed on preference ${preference} using instancetype u1.medium"
         exit 1
     fi
+
+    if ! ${KUBECTL} wait --for=condition=Ready vmis/"vm-${preference}" ; then
```

Comment:
What is the default timeout?

Reply:
`30s` I believe.

Reply:
Isn't too low? IMHO 1 or 2 mins sounds reasonable

Reply:
`30s` should be plenty.